pcdsdevices/mirror.py

```diff
@@ -915,3 +915,26 @@ class XOffsetMirrorState(XOffsetMirror, CoatingState):
     """
     # UI representation
     _icon = 'fa.minus-square'
+
+
+class OpticsPitchNotepad(BaseInterface, Device):
+    """
+        Class for storing pitch positions based on state 
+    """
```

Comment:
Docstring formatting
```suggestion
    """
    Class for storing pitch positions based on state.
    """
```
You should also add what this is.
What is the optics notepad?
What is it used for?
Why does it exist?
How is this different from the PV setpoints on each IOC?